docs/source/release_notes/lts/next.rst

```diff
@@ -88,6 +88,118 @@ introduced by the next LTS version are:
+  "non-Pythonic locations" such as ``/etc`` are no longer attempted by
+  the Python ``setup.py`` code.  Configuration files, for instance,
+  are now considered package data files of the ``avocado`` package.
+  The end result is that installation from source works fine outside
+  virtual environments (in addition to installations *inside* virtual
+  environments).
+
+
+* A :ref:`new plugin <glib-plugin>` enables users to list and execute
+  tests based on the `GLib test framework
+  <https://developer.gnome.org/glib/stable/glib-Testing.html>`_.  This
+  plugin allows individual tests inside a single binary to be listed
+  and executed.
+
+* Users of the YAML test loader have now access to a few special keys
+  that can tweak test attributes, including adding prefixes to test
+  names.  This allows users to easily differentiate among execution of
+  the same test, but executed different configurations.  For more
+  information, look for "special keys" in the :ref:`YAML Loader plugin
+  documentation <yaml_loader>`.
+
+* Users can now dump variants to a (JSON) file, and also reuse a
+  previously created file in their future jobs execution.  This allows
+  users to avoid recomputing the variants on every job, which might
+  bring significant speed ups in job execution or simply better
+  control of the variants used during a job.  Also notice that even
+  when users do not manually dump a variants file to a specific
+  location, Avocado will automatically save a suitable file at
+  ``jobdata/variants.json`` as part of a Job results directory
+  structure.
+
+* SIMPLE tests were limited to returning PASS, FAIL and WARN statuses.
+  Now SIMPLE tests can now also return SKIP status.  At the same time,
+  SIMPLE tests were previously limited in how they would flag a WARN
+  or SKIP from the underlying executable.  This is now configurable by
+  means of regular expressions.
+
+* The :mod:`avocado.utils.process` has seen a number of changes
+  related to how it handles data from the executed processes.  In a
+  nutshell, process output (on both ``stdout`` and ``stderr``) is now
+  considered binary data.  Users that need to deal with text instead,
+  should use the newly added
+  :attr:`avocado.utils.process.CmdResult.stdout_text` and
+  :attr:`avocado.utils.process.CmdResult.stderr_text`, which are
+  convenience properties that will attempt to decode the ``stdout`` or
+  ``stderr`` data into a string-like type using the encoding set, and
+  if none is set, falling back to the system default encoding.  This
+  change of behavior was needed to accommodate Python's 2 and Python's
+  3 differences in bytes and string-like types and handling.
+
+* The TAP result format plugin received improvements, including
+  support for reporting Avocado tests with CANCEL status as SKIP
+  (which is the closest status available in the TAP specification),
+  and providing more visible warning information in case Avocado tests
```

Comment:
how about mentioning the `comment` in the sentence? I mean that is how warnings are passed.

Reply:
Done.